tools/codegen/matter_idl_plugin/matter_js_schema.jinja

```diff
@@ -0,0 +1,79 @@
+// AUTO-GENERATED with matter_idl.
+/**
+ * @license
+ * Copyright 2022 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AccessLevel, Attribute, Cluster, Command, TlvNoArguments, WritableAttribute } from "./Cluster.js";
+import { MatterCoreSpecificationV1_0 } from "../spec/Specifications.js";
+import { TlvField, TlvObject } from "../tlv/TlvObject.js";
+import { TlvEnum, TlvUInt16, TlvUInt64 } from "../tlv/TlvNumber.js";
+import { TlvString } from "../tlv/TlvString.js";
+import { TypeFromSchema } from "../tlv/TlvSchema.js";
+import { TlvBoolean } from "../tlv/TlvBoolean.js";
+
+{%- if cluster.enums %}
+
+  // Enums
+{%-   for entry in cluster.enums %}
+export const enum {{entry.name}} {
```

Comment:
should we export by default?

Reply:
Looks like enums in hand-written clusters are all exported, so yes.